jenkinsfiles/stable

```diff
@@ -123,6 +123,70 @@ pipeline {
             }
         }
 
+        stage('Update IM Play instance') {
+            environment {
+                HTTP = "http --check-status"
+                IM_REPO_URL = "https://github.com/dhis2-sre/im-manager"
+                IM_HOST = "https://api.im.dhis2.org"
+                INSTANCE_URL = "https://play.im.dhis2.org/stable"
+                IMAGE_REPOSITORY = "core"
+                IMAGE_PULL_POLICY = "Always"
+                FLYWAY_MIGRATE_OUT_OF_ORDER = "true"
+                FLYWAY_REPAIR_BEFORE_MIGRATION = "true"
+                INSTANCE_TTL = "315360000"
+                STARTUP_PROBE_FAILURE_THRESHOLD = "50"
+                LIVENESS_PROBE_TIMEOUT_SECONDS = "3"
+                READINESS_PROBE_TIMEOUT_SECONDS = "3"
+            }
+
+            steps {
+                script {
+                    if(env.DOCKER_IMAGE_TAG.endsWith("rc")) {
+                        catchError(buildResult: 'SUCCESS', stageResult: 'NOT_BUILT') {
+                            error "Skipping..." // Force an error that we will catch to set the stageResult
```

Comment:
If you want to skip this stage when the `env.DOCKER_IMAGE_TAG.endsWith("rc")`, you can use the [`when` directive](https://www.jenkins.io/doc/book/pipeline/syntax/#when).

```
when {
                expression { env.DOCKER_IMAGE_TAG.endsWith("rc") }
}
```